google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/standard/AppEngineStandardFacetType.java

```diff
@@ -35,10 +36,12 @@
  */
 public class AppEngineStandardFacetType
     extends FacetType<AppEngineStandardFacet, AppEngineStandardFacetConfiguration> {
+
+  public static final FacetTypeId<AppEngineStandardFacet> ID = new FacetTypeId<>("appEngine");
```

Comment:
probably, but I'm hesitant to change it here because it could affect previously created facets on existing projects